easy_login/tests/tests.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import factory
+from django.contrib.auth.models import User
+from django.test import RequestFactory, TestCase, Client
+from bs4 import BeautifulSoup
+from django.urls import reverse_lazy
+
+from easy_login.context_processors import easy_login
+
+import re
+
+
+class AdminFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', )
+
+    username = 'Admin'
+    email = 'admin@adm.com'
+    is_staff = True
+    is_superuser = True
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
```

Comment:
you create two the same classes.
Factory can accept params  UserFactory(is_superuser)
also, it's can automate generate users
and create the folder with factories.